french/dessine/miel/forest/fuite/fuite.md

```diff
@@ -0,0 +1 @@
+Je decide de me sauver de l'ours en courrant vers la porte par laquelle je suis passe. Apres quelques pas l'ours m'a rattrape et se delecte du miel...
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.